neqo-transport/src/connection/tests/cc.rs

```diff
@@ -211,12 +210,11 @@ fn single_packet_on_recovery() {
     assert!(dgram.is_some());
 }
 
-#[test]
 /// Verify that CC moves out of recovery period when packet sent after start
 /// of recovery period is acked.
-fn cc_cong_avoidance_recovery_period_to_cong_avoidance() {
-    let mut client = default_client();
-    let mut server = default_server();
+fn cc_cong_avoidance_recovery_period_to_cong_avoidance(cc_algorithm: CongestionControlAlgorithm) {
```

Comment:
This is the only test in `cc.rs` that specifically depends on the underlying `CongestionControlAlgorithm` used. The other tests focus on the high level `ClassicCongestionControl`.

For the above reason, I think it is worth running this unit test with both `NewReno` and `Cubic`.

Previously this test would assert on the concrete individual `NewReno` window increases. That is difficult to do when supporting two `CongestionControlAlgorithm`s.

Instead I suggest simply asserting that after x rounds, the congestion controller reaches the congestion window before the loss event again. Thus it switched from `Recovery` to `CongestionAvoidance`.